DataStructuresMemories/Models/DataStructuresModel.swift

```diff
@@ -60,15 +60,15 @@ class DataStructModel: DataStruct {
     func getDescription(of structure: String) -> String {
         guard let currentDataStruct = DataStructuresNames(rawValue: getCurrentIndex(of: structure))
         else {
-            fatalError("Name: \(structure) does not match any name of: DataStructuresNames")
+            fatalError("Name: \(structure) does not match any name of: DataStructuresNames.")
```

Comment:
Wow. would you kill baby who doesn't say you hello? 
fatalError is from another tale, please avoid using it in real projects

Reply:
Got it)